### PR TITLE
Search

### PR DESCRIPTION
**Pull request by mohanagy**

### CODE REVIEW

File: public/src/components/createTrip/createTrip.jsx [hunk 1]
```diff
@@ -134,21 +130,12 @@ class CreateTrip extends React.Component{
...
-                  </div>
-                  <div className='form-group'>
-                    <label>Pickup time</label>
-                    <input
-                      type='time'
-                      value={this.state.pass_point_time}
+                      value={this.state.details}
                       className='form-control'
-                      onChange={this.changePassingPointTime.bind(this)}
+                      onChange={this.changeDetails.bind(this)}
```
Comment (saralk) [inline public/src/components/createTrip/createTrip.jsx]: make this a text area

File: public/src/components/createTrip/createTrip.jsx [hunk 2]
```diff
@@ -134,21 +130,12 @@ class CreateTrip extends React.Component{
...
                       />
```
Comment (saralk) [inline public/src/components/createTrip/createTrip.jsx]: Add some help text, something like

"e.g. 'Meet outside the National Hotel'"

File: scripts/sql.js [hunk 3]
```diff
@@ -28,8 +28,7 @@ const trip = `CREATE TABLE IF NOT EXISTS trip (
     location_to_id INT references location(location_id),
     time varchar(30),
     date varchar(30),
-    pass_point_time varchar(30),
-    passing_by varchar(50),
+    details varchar(255),
```
Comment (saralk) [inline scripts/sql.js]: this wont work on an already existing database, look at line 42 of this file for en example of what to do instead.

Comment (mohanagy) [inline scripts/sql.js]: actually we  modified the exist database to be compatible with the new changes  :grimacing:

File: public/src/components/createTrip/createTrip.jsx (hunk 1 above)
Comment (mohanagy) [inline public/src/components/createTrip/createTrip.jsx]: [Fixed](https://github.com/gazaskygeeks/crossing-tracker/pull/114/commits/ab35ae5a9abc1761894daedff6fc101008f35d4f#diff-3430c62448f448ef57a15c6f6cf9abb6)

File: public/src/components/createTrip/createTrip.jsx (hunk 2 above)
Comment (mohanagy) [inline public/src/components/createTrip/createTrip.jsx]: [fixed](https://github.com/gazaskygeeks/crossing-tracker/pull/114/commits/ab35ae5a9abc1761894daedff6fc101008f35d4f#diff-3430c62448f448ef57a15c6f6cf9abb6)

File: scripts/sql.js (hunk 3 above)
Comment (saralk) [inline scripts/sql.js]: ok, but don't do this in the future!

**Pull request merged**